solr/core/src/test/org/apache/solr/handler/admin/AdminHandlersProxyTest.java

```diff
@@ -91,6 +98,32 @@ public void proxyMetricsHandlerAllNodes() throws IOException, SolrServerExceptio
     assertNotNull(((NamedList)nl.get(nl.getName(1))).get("metrics"));
   }
 
+  @Test
+  public void proxyLoggingHandlerAllNodes() throws IOException, SolrServerException {
+    CollectionAdminRequest.createCollection("collection", "conf", 2, 2).process(solrClient);
+
+    mparams = new ModifiableSolrParams();
```

Comment:
BTW, Solr test infrastructure (SolrTestCaseJ4) defines a params(...) method that is more succinct.  This is fine though!